tests/test_views/test_api_view.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from aiohttp import web, hdrs
+from marshmallow import Schema, fields
+
+from colibris.middleware.errors import handle_errors_json
+
+from colibris.views import APIView
+
+
+class ItemSchema(Schema):
+    name = fields.String()
+    info = fields.String()
+    count = fields.Integer()
+
+
+class ItemsView(APIView):
```

Comment:
@ccrisan, what do you think about `APIView` ? This is a basic example of how it can be used.

Reply:
@lucifurtun almost OK. I don't like that we lose the specificity of `HTTPSchemaValidationError` and use a simple invalid request. I believe schema validation should have a more specific code and details, as `HTTPSchemaValidationError` offered.

Also, we should remove the middleware leftovers if we go this way.

Reply:
Ok, I'll try to integrate `HTTPSchemaValidationError` in the proposed way to handle errors. Do you see other middleware leftovers other than this one?